daemon/containerd/imagespec.go

```diff
@@ -109,7 +109,6 @@ func dockerOCIImageConfigToContainerConfig(cfg imagespec.DockerOCIImageConfig) *
 	for k, v := range cfg.ExposedPorts {
 		exposedPorts[nat.Port(k)] = v
 	}
-
```

Comment:
If you're gonna update this PR (for other review comments) can you remove this change from the first commit, so that this file disappears from the final diff? 🤗

Reply:
oh yes please! I didn't leave a comment because it was such a small thing and wouldn't want you to have to rebase to address just that, but I'd be happy if it happened 😅

Reply:
Done!